index.php

```diff
@@ -60,56 +59,50 @@
 $oversize = $DB->get_field_sql("SELECT COUNT(*)
                                  FROM {linkchecker_url}
                                 WHERE filesize > ?",  $opts );
-?>
 
-<table>
-    <tr>
-        <td><?php echo get_string('botuser', 'local_linkchecker_robot'); ?></td>
-        <td><?php echo $boterror ? $boterror : 'Good'; ?></td>
-    </tr>
-    <tr>
-        <td><?php echo get_string('curcrawlstart', 'local_linkchecker_robot'); ?></td>
-        <td><?php echo $crawlstart ? userdate( $crawlstart) : 'Never run'; ?></td>
-    </tr>
-    <tr>
-        <td><?php echo get_string('lastcrawlend', 'local_linkchecker_robot'); ?></td>
-        <td><?php echo $crawlend ? userdate( $crawlend) : 'Never finished'; ?></td>
-    </tr>
-    <tr>
-        <td>
-            <?php echo get_string('lastcrawlproc', 'local_linkchecker_robot'); ?></td>
-        <td><?php echo $crawltick ? userdate( $crawltick) : '-'; ?></td>
-    </tr>
-    <tr>
-        <td><?php echo get_string('lastqueuesize', 'local_linkchecker_robot'); ?></td>
-        <td><?php echo $oldqueuesize ?></td>
-    </tr>
-    <tr>
-        <td><?php echo get_string('queued', 'local_linkchecker_robot'); ?></td>
-        <td><a href="report.php?report=queued"><?php echo $queuesize; ?></a></td>
-    </tr>
-    <tr>
-        <td><?php echo get_string('recent', 'local_linkchecker_robot'); ?></td>
-        <td><a href="report.php?report=recent"><?php echo $recent; ?></a></td>
-    </tr>
-    <tr>
-        <td><?php echo get_string('broken', 'local_linkchecker_robot'); ?></td>
-        <td><a href="report.php?report=broken"><?php echo $broken; ?></a></td>
-    </tr>
-    <tr>
-        <td><?php echo get_string('oversize', 'local_linkchecker_robot'); ?></td>
-        <td><a href="report.php?report=oversize"><?php echo $oversize; ?></a></td>
-    </tr>
-</table>
+$table = new html_table();
+$table->head = array(get_string('robotstatus', 'local_linkchecker_robot'));
+$table->headspan = array(2, 1);
+$table->data = array(
+    array(
+        get_string('botuser', 'local_linkchecker_robot'),
+        $boterror ? $boterror : 'Good'
```

Comment:
you have fixed this already, was there more commits you nee to squash into this branch?